xml/art-nvidia-vgpu.xml

```diff
@@ -466,18 +466,18 @@ lrwxrwxrwx 1 root root           0 Sep 21 11:58 virtfn9 -> ../0000:b1:01.5
 <screen>
 &prompt.user;cd /sys/bus/pci/devices/0000:b1:00.0/virtfn1/mdev_supported_types
 &prompt.user;for i in *; do echo "" $(cat $i/name) available: $(cat $i/avail*); done
```

Comment:
Please add '$' in the echo statement. It is required.

`for i in *; do echo "$i" ....`